fixtures/data_criteria/care_goal.json

```diff
@@ -0,0 +1,17 @@
+    {
+        "title": "Repurposed Symptom Active for Testing",
```

Comment:
This json file and the ones below it all have some inconsistent indentation combined with uneven braces. I would prefer pure TABS to avoid this issue, but at least fix the files and make them consistent. It's hard to follow what fields are in what objects with inconsistent style.